__tests__/app.test.js

```diff
@@ -226,6 +226,25 @@ describe("POST 201: /api/articles/:article_id/comments", ()=>{
   
 })
 
+describe("DELETE 204: /api/comments/:comment_id", ()=>{
+  test("DELETE 204: deletes the selected comment and returns 204 status code", () =>{
+    return request(app).delete("/api/comments/1").expect(204)
+  })
```

Comment:
Nice testing! You could potentially add a .then here and check the body is empty, but I'll leave it up to you 🙂